src/Workspaces/Core/Portable/CodeFixesAndRefactorings/DefaultFixAllProviderHelpers.cs

```diff
@@ -39,7 +45,7 @@ internal static class DefaultFixAllProviderHelpers
             return null;
 
         return CodeAction.Create(
-            title, c => Task.FromResult(solution));
+            title, _ => Task.FromResult(solution));
```

Comment:
@mavasani Aside: why do we do all teh expensive comptuation above, instead of doing it in the async callback passed to CodeAction.Create?